f5_sphinx_theme/static/css/f5-theme.css

```diff
@@ -318,6 +321,25 @@ a[aria-expanded="true"]::before {
 #search-results > ul {
   display: block;
 }
+
+/*
+ * Version Warning
```

Comment:
This is for a feature that we may not end up doing but I'll just leave it in for now anyway....